cmd/kube-scheduler/app/server.go

```diff
@@ -99,6 +98,8 @@ for more information about scheduling and the kube-scheduler component.`,
 			return nil
 		},
 	}
+	cmd.SetGlobalNormalizationFunc(pflag.CommandLine.GetNormalizeFunc())
```

Comment:
this is covered in cli.Run now, right?

Reply:
Yes. I'll remove it here.

Reply:
... and also in the other `New*Command` functions.